providers/nfs_provider/deploy/nfs-provider-deployment.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 The SODA Authors.
```

Comment:
Can we arrange all nfs server deployment things together to one yaml so that that can be setup first and then install kahu